.github/workflows/serverless-artillery-action-users-id-api.yml

```diff
@@ -0,0 +1,44 @@
+name: Performance-Test serverless-artillery-action-users-id-api
+on: [push]
+    
+jobs:
+  slsart-perf:
+    name: Serverless Artillery Users API Test Performance testing
+    runs-on: ubuntu-latest
+    env:
+      STAGE_NAME: Test-userid-api
+    steps:
+    - name: Checkout Code
+      uses: actions/checkout@v2
+
+    - name: Setup Node.js
+      uses: actions/setup-node@v2
+      with:
+        node-version: '14'
+        check-latest: true
+    - name: Configure AWS Credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: us-west-2
+      
+    - name: Install serverless and serverless-artillery
+      run: |
+        npm install -g serverless@1.38.0 
+        npm install -g serverless-artillery
+
+    - name: Deploy functions
+      run: slsart deploy --stage $STAGE_NAME --region=us-west-2
+    
+    - name: invoke functions
+      run: slsart invoke -o test/serverless-artillery/serverless-artillery-test-users-id-api.yaml --region us-west-2 --stage $STAGE_NAME --jsonOnly > test/results/slsart-serverless-artillery-test-users-id-api-results.json
```

Comment:
The output should match our naming conventions for the json file: [bucketname]/YYYY/mm/DD/HH-MM-ss.xxx.json

Reply:
I fixed the naming conventions